cluster/environments/azure-multiple-clusters/aks-eastus.tf

```diff
@@ -13,60 +13,68 @@ locals {
   east_ip_address_out_filename = "${local.east_prefix}_ip_address"
 }
 
-# Creates vnet
+# Creates east vnet
 module "east_vnet" {
+  # source = "github.com/Microsoft/bedrock/cluster/azure/vnet"
   source = "../../azure/vnet"
 
   resource_group_name     = "${local.east_rg_name }"
   resource_group_location = "${local.east_rg_location}"
   subnet_names            = ["${var.cluster_name}-aks-subnet"]
   address_space           = "${var.east_address_space}"
   subnet_prefixes         = "${var.east_subnet_prefixes}"
-
   tags = {
     environment = "azure-multiple-clusters"
   }
 }
 
-# Creates aks cluster
-module "east_aks" {
-  source = "../../azure/aks"
+data "azurerm_client_config" "eastclient" {}
```

Comment:
As commented above, we can assume the environment is deployed in a single subscription so we can move the `azurerm_client_config` data provider to main.tf to reuse in all files.